modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -65,6 +69,7 @@
     protected String modelFileSuffix = "";
     protected String fileNaming = "camelCase";
     protected Boolean stringEnums = false;
+    protected String queryParamObjectFormat = "dot";
```

Comment:
Suggest using `enum` instead of hardcoded strings. See e.g. `MODEL_PROPERTY_NAMING_TYPE`

Reply:
Would the enum constant go with `MODEL_PROPERTY_NAMING_TYPE` in the same global file or be held locally?  (I'll do the later for now)

Reply:
I believe it should be defined locally, as it's very specific to the angular generator.

Reply:
Thanks.  One more query if you don't mind!  If `this.queryParamObjectFormat` is an enum then how do we approach the following?  It was a string but with enum we need a bit more, or shall I just omit the CLI option?

```
this.cliOptions.add(new CliOption( // ... )
.defaultValue(this.queryParamObjectFormat));
```

Reply:
`.defaultValue(this.queryParamObjectFormat.name())`
you could refer to how `MODEL_PROPERTY_NAMING` and `ENUM_PROPERTY_NAMING` are used inside `AbstractTypeScriptClientCodegen`

Reply:
@amakhrov Thank you for the guidance.  Applied feedback.